app/browsered.js

```diff
@@ -7,3 +7,4 @@ const multiSelects = require('./client-side-scripts/multi-select')
 window.$ = window.jQuery = $
 
 multiSelects.enableMultiSelects()
+module.exports.fieldValidation = require('./browsered/field-validation')
```

Comment:
I find it's generally better to export a function that has to be explicitly called rather than have a module change the global environment just by being required.
Only my opinion though.

Reply:
Ok cool will change